readthedocs/projects/tasks.py

```diff
@@ -1393,3 +1394,17 @@ def finish_inactive_builds():
         'Builds marked as "Terminated due inactivity": %s',
         builds_finished,
     )
+
+
+@app.task
```

Comment:
I suppose that you should pass `queue='web'` since we are accessing the database from inside the task.